common/workunit/workunit.cpp

```diff
@@ -4611,6 +4611,34 @@ IStringVal &getProcessQueueNames(IStringVal &ret, const char *process, const cha
 #define ECLSCHEDULER_QUEUE_EXT ".eclscheduler"
 #define ECLAGENT_QUEUE_EXT ".agent"
 
+extern WORKUNIT_API IStringVal &getDFUServerQueueNames(IStringVal &ret, const char *process, const char *propName)
+{
+    Owned<IEnvironmentFactory> factory = getEnvironmentFactory();
+    Owned<IConstEnvironment> env = factory->openEnvironment();
+    if (!env)
+        return ret;
+
+    VStringBuffer pName("@%s", (propName && *propName) ? propName: "queue");
```

Comment:
I can't see anywhere that propname (qtype) is used, or think of any use cases..
For a DFUServer the queue name will always be called @queue.

So I think you should remove this code, 'propname' and associated code.